CHANGES.md

```diff
@@ -1,3 +1,22 @@
+## edge-20.3.3
+
+This release introduces new experimental CLI commands for querying metrics using
+the Service Mesh Interface (SMI) and for multi-cluster support via service
```

Comment:
Maybe use `.. as well as ..` instead of `.. and for ..`?

Reply:
hmm, i'm not sure..."new experimental CLI commands for querying ... as well as multi-cluster support ..." makes it sound like "multi-cluster support via service mirroring" is added in this release, rather than "_new experimental CLI commands for_ multi-cluster support via service mirroring"

Reply:
Yep I see what you mean. Reading through it the first time caught my attention, but definitely doesn’t need to be changed.